packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -631,12 +631,6 @@ export function TokenSearch({
         })
       }
 
-      // do not allow import of withdraw-only tokens at deposit mode
-      if (isDepositMode && isWithdrawOnlyToken(_token.address, childChain.id)) {
-        openTransferDisabledDialog()
-        return
-      }
```

Comment:
Moved to the `TransferDisabledDialog` because `isWithdrawOnlyToken` is now an async method and it's better to depend on the `selectedToken` state than the `_token` var passed into this function synchronously